tests/queries/0_stateless/02327_capnproto_protobuf_empty_messages.sh

```diff
@@ -5,10 +5,10 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-USER_FILES_PATH=$(clickhouse-client --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
+USER_FILES_PATH=$($CLICKHOUSE_CLIENT_BINARY --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
```

Comment:
We usually use just `CLICKHOUSE_CLIENT`, not `CLICKHOUSE_CLIENT_BINARY`

Reply:
Thank you @mkmkme !

Reply:
@Avogar so I made a small investigation and realised that `CLICKHOUSE_CLIENT` is not really applicable there.

So,
1. `CLICKCLOUSE_CLIENT_BINARY` is just a path to ClickHouse that has been passed via `-b` or `clickhouse` from `$PATH` plus `-client`. 
2. `CLICKHOUSE_CLIENT` is `CLICKHOUSE_CLIENT_BINARY` with all the options
3. On my setup `CLICKHOUSE_CLIENT_BINARY="/home/parallels/clickhouse client"`, and `CLICKHOUSE_CLIENT` can be found [here](https://pastila.nl/?00132659/a7070a5579b0cfc6a8952e5008b42dde#zOJP1bZc5IQIZpru2hhTDA==)

When I call 
```bash
$CLICKHOUSE_CLIENT_BINARY --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception
```

I get

```
Code: 107. DB::Exception: Received from localhost:9000. DB::Exception: File /home/parallels/clickhouse-rundir/user_files/nonexist.txt doesn't exist. (FILE_DOESNT_EXIST)
```

The testcase extracts the path from it via `awk '{gsub("/nonexist.txt","",$9); print $9}'`

When I call
```bash
$CLICKHOUSE_CLIENT --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception
```

I get a different exception:

```
Code: 81. DB::Exception: Received from localhost:9000. DB::Exception: Database test_s1k3cu9o does not exist. (UNKNOWN_DATABASE)
```

Which breaks the testcase flow.

That said, it seems that some of the arguments passed to `CLICKHOUSE_CLIENT` actually changes its behaviour, therefore replacing `clickhouse-client` with `CLICKHOUSE_CLIENT_BINARY` seems more appropriate.

Reply:
Well, `UNKNOWN_DATABASE ` exception is expected if you call this command without using `clickhouse-test`, because `clickhouse-test` creates this database. And in `clickhouse-test` we add all these additional options to `CLICKHOUSE_CLIENT`, but non of them can lead to a problem with query `select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')`

Reply:
>none of them can lead to a problem with query select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')

Well it seems that this statement is incorrect, because it clearly causes issues on my side. Additionally, that test created files with names `0x00000000074f784c` and `executeQuery:`

Reply:
> Well it seems that this statement is incorrect, because it clearly causes issues on my side

Well, issues on your side can also mean you do something wrong. I don't see any possible problems with using `CLICKHOUSE_CLIENT`, and we do it in one test: `02455_one_row_from_csv_memory_usage.sh`.
How do you run tests? The correct way is to run `clickhouse-test <test_name>`, and most likely no problem will happen.

Reply:
>I don't see any possible problems with using CLICKHOUSE_CLIENT, and we do it in one test

Have you tried it in this particular test? Did it work for you?

>How do you run tests? 

`./clickhouse-test -b ~/clickhouse 02327_capnproto` from `tests` directory.  `~/clickhouse` is a binary that I got via `curl https://clickhouse.com | sh`. The same binary is running as the server on the same machine.

Reply:
I tried this test, and it works: https://pastila.nl/?00154a89/28ff8704fb7e3c4e4f43c6f807d12a1a#rCiZ7DfPSwXtzabSSKeyZA==

Reply:
Ah, ok, let me delete path to my binaries from the PATH and try again

Reply:
Wait, I tried it as well and it worked. But it breaks when I change the second query to just use `$CLICKHOUSE_CLIENT`

Reply:
Seems like the problem is in client setting `send_logs_level=warning`, with it all exceptions are printed with stacktrace, so we may incorrectly extract path to a file from it. But I don't know why we print stacktraces when we set `send_logs_level`, maybe there is a bug actually